helix-view/src/info.rs

```diff
@@ -17,6 +17,13 @@ pub struct Info {
 
 impl Info {
     pub fn new(title: &str, body: Vec<(String, String)>) -> Self {
+        let item_width = body.iter().map(|(item, _)| item.width()).max().unwrap();
```

Comment:
Undo this please. A whole bunch of code would now be executed even if text is empty

Reply:
sorry, I fixed